Packages/vcs/Lib/vcsvtk/vectorpipeline.py

```diff
@@ -27,14 +30,45 @@ def plot(self, data1, data2, tmpl, grid, transform):
         data1 = self._context().trimData2D(data1)
         data2 = self._context().trimData2D(data2)
 
+        scale = 1.0
+
+        lat = data1.getLatitude()[:]
```

Comment:
data1 could be `NoneType` 

`vcs_no_xtra_elts` fails because of that. (https://open.cdash.org/testDetails.php?test=375905204&build=4024522)